frontend_app_kanban/src/debug.js.sample

```diff
@@ -18,5 +18,6 @@ export const debug = {
     content_id: 1,
     content_type: 'kanban',
     workspace_id: 1
-  }
+  },
+  workspaceId: 1
```

Comment:
this confuses me
the space id has always been taken from the content

Reply:
It is used by `PopupCreateKanban` which is similar to `PopupCreateFile` regarding this aspect. Those popups cannot have a real content as their job is to create one.
But maybe I'm the first to write a unit test for one of those popups.

Reply:
@grignards up